tests/RobotFramework/tests/cumulocity/log/log_operation.robot

```diff
@@ -101,6 +101,7 @@ Default plugin configuration
     Stop Service    tedge-agent
     ${timestamp}=        Get Unix Timestamp
     Start Service    tedge-agent
+    Sleep    5s    reason=Added waiting time because this test sometimes fails when running as part of workflow
```

Comment:
I'm not sure how this helps exactly, as the "activating" state is something part of systemd, and the service status is sent by the tedge-agent process.

So if this is flaky for other reasons, then it might be better to check that the service status is "down? prior to start the service again.

Reply:
Actually it is not "flaky", it is failing always when this test is part of "bigger run" but never failing when run as only one test or when part of not so big (don't know the exact boundry number) run. After adding the sleep time it never failed again

Reply:
But then why does the CI pass?

Reply:
But the sleep is not logical for me (despite the effect)...It would be more correct to establish that service status is down (after doing `Stop Service`), then proceeding to start the service...in that case you then don't need an explicit sleep.

Reply:
added check if sevice is down solved the issue about test fail.